demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1640,6 +1640,10 @@ export class DemoMeetingApp
       case 'meetingReconnected':
       case 'receivingAudioDropped':
       case 'signalingDropped':
+      case 'sendingAudioFailed':
+      // We can move sendingAudioRecovered below to include meetingHistory attribute if we re-use it for
```

Comment:
Lol @ leaving a TODO but not marking it a TODO since we aren't supposed to do that i guess :P.

Reply:
:sweat_smile: Well it's supposed to serve like a note to the person implementing the `sendingAudioSilenced` event but I guess i could track it offline